lib/common/src/Extension.php

```diff
@@ -10,18 +10,35 @@
 interface Extension
 {
 
+    const AD                  = 'amp-ad';
     const ANALYTICS           = 'amp-analytics';
+    const ANIM                = 'amp-anim';
     const AUDIO               = 'amp-audio';
     const BIND                = 'amp-bind';
+    const BRIGHTCOVE          = 'amp-brightcove';
+    const DAILYMOTION         = 'amp-dailymotion';
+    const DELIGHT_PLAYER      = 'amp-delight-player';
     const DYNAMIC_CSS_CLASSES = 'amp-dynamic-css-classes';
     const EXPERIMENT          = 'amp-experiment';
+    const FACEBOOK            = 'amp-facebook';
     const GEO                 = 'amp-geo';
+    const GFYCAT              = 'amp-gfycat';
+    const IFRAME              = 'amp-iframe';
+    const IMAGE               = 'amp-img';
```

Comment:
I think we should stick with the extension name being the same as the constant name (other areas where this is being used would also need to be updated).

```suggestion
    const IMG                 = 'amp-img';
```

Reply:
I will do this in a separate PR.

Reply:
=> https://github.com/ampproject/amp-toolbox-php/issues/5